src/easydmp/dmpt/models.py

```diff
@@ -2197,6 +2230,14 @@ def validate_choice(self, data):
         return False
 
 
+class DoSectionQuestion(NoCheckMixin, Question):
```

Comment:
`OptionalSectionQuestion` probably. Inheriting from `BooleanQuestion` if possible.

Reply:
With my class name it matches what yes/no means, but...